### PR TITLE
Add elasticsearch6 container

### PR DESCRIPTION
**Pull request by barrucadu**
search-api is trying to connect to this

### CODE REVIEW

**Comment by barrucadu** [comment]
This is now failing for the same reason as all the other e2e tests (chrome issues) rather than failing because the container is missing.

File: docker-compose.yml [hunk 1]
```diff
@@ -83,12 +83,25 @@ services:
...
 
+  elasticsearch6:
+    image: elasticsearch:6.7.0
+    environment:
+      - http.host=0.0.0.0
+      - transport.host=127.0.0.1
+      - xpack.security.enabled=false
+    healthcheck:
+      << : *default-healthcheck
+      test: "curl --silent --fail localhost:9200/_cluster/health || exit 1"
```
Comment (bilbof) [inline docker-compose.yml]: Will it be at `localhost:9200`? Sorry I might have misunderstood this.

Comment (barrucadu) [reply docker-compose.yml]: Inside the container it's `localhost:9200`, outside the container it's `elasticsearch6:9200`

**Comment by barrucadu** [comment]
Going to disable the branch protection for this as this is slightly less broken than `test-against` currently is

**Pull request merged**